packages/viewer/src/scenes/operations.ts

```diff
@@ -74,6 +87,18 @@ export class SceneOperationBuilder
     );
   }
 
+  public select(color: ColorMaterial): SceneOperationBuilder {
```

Comment:
I think we can do this in a follow up, but I think we should have a way to set the default selection color on the viewer, then making this arg optional.

Reply:
Yeah, that makes sense to me. It's definitely a bit messy currently to try to manage that color across all cases of selection when consuming the viewer. I'll merge this change for now and follow-up with a change to support that